docs/theming/developers.md

```diff
@@ -44,13 +44,108 @@ and elements. In other words, theming is the developer’s process of
 orchestrating design tokens with elements, particularly by way of themeable 
 container elements.
 
+## How theming works
+
```

Comment:
I'd love to see something along these lines with the corresponding CSS alongside it (expanded out): 
```
As an example, here is the CSS of a theme we'll call "bordeaux". We set some initial base color scheme values for the theme, and then use those to set the appropriate design system values...
```